### DISCUSSION TITLE
jquery引入js后方法调用不到

### DISCUSSION THREAD

**Discussion by ghost** [umijs/qiankun#2690] [Q&A]
js引入成功后，在调用的时候报错
<img width="672" alt="image" src="https://github.com/umijs/qiankun/assets/73871391/d6b15983-299c-44d4-96be-b138266aa189">
![image](https://github.com/umijs/qiankun/assets/73871391/a2531574-70b7-4409-ba52-cb8aa38278f3)
![image](https://github.com/umijs/qiankun/assets/73871391/b7286422-d4e1-45b7-aef7-6cc48f7da929)
![image](https://github.com/umijs/qiankun/assets/73871391/5d4b0cdc-37b8-4580-8eda-17aef8f0d8e6)

**Comment by gongshun** [comment]
https://github.com/umijs/qiankun/issues/1894
